src/main/kotlin/org/radarbase/redcap/integration/Integrator.kt

```diff
@@ -57,33 +57,31 @@ class Integrator(
             throw IllegalRequestException("Some of the required values were missing.")
         }
 
-        keys.add(IntegrationData.SUBJECT_ID_LABEL)
-        logger.info("Attribute Keys: {}", keys.toTypedArray())
-        val result: MutableMap<String, String> = try {
-            redCapIntegrator.pullFieldsFromRedcap(keys, recordId)
+        try {
+            attributes = redCapIntegrator.pullFieldsFromRedcap(redCapInfo.attributes?.map { a -> a.fieldName }!!, recordId)
```

Comment:
It might be a good improvement to have a configurable event name with each attribute in the config and then only query the attributes for those events to avoid unnecessary conflicts.

Reply:
I've added it as an issue here for now: #72